src/controllers/userProfileSettings.js

```diff
@@ -0,0 +1,47 @@
+import bcrypt from 'bcrypt';
+// import Jwt from 'jsonwebtoken';
+import User from '../models/user.model.js';
+import redisClient from '../helpers/redis.js';
+
+const { hash } = bcrypt;
+
+const UpdateUserInfo = async (req, res) => {
+  const userId = req.user.id;
+  const {
+    username,
+    // email,
```

Comment:
Why is email commented out ?

Reply:
here email is not to be changed

Reply:
simply, i am not even allowed to change user's email, password, and username